clients/admin-ui/cypress/e2e/privacy-experiences.cy.ts

```diff
@@ -3,8 +3,8 @@ import { stubPlus } from "cypress/support/stubs";
 import { PRIVACY_EXPERIENCE_ROUTE } from "~/features/common/nav/v2/routes";
 import { RoleRegistryEnum } from "~/types/api";
 
-const EXPERIENCE_ID = "pri_0338d055-f91b-4a17-ad4e-600c61551199";
-const DISABLED_EXPERIENCE_ID = "pri_8fd9d334-e625-4365-ba25-9c368f0b1231";
```

Comment:
These IDs correspond to the experiences in the `privacy-experiences/list.json` fixtures so changing these to row indices will cause some issues when we remove the skips you added in this PR.

This feature is working locally so I don't want to hold it up but if you have time I would move away for using indices for the assertions and use a more unique value in the table rows to verify functionality.